File: zero/Maruko.Zero/Internal/Domain/SysOperate.cs
using FreeSql.DataAnnotations;
using Maruko.Core.FreeSql.Internal;

namespace Maruko.Zero
{
    [Table(Name = "sys_operate")]
    public class SysOperate : FreeSqlEntity
    {
        [Column(Name = "name", StringLength = 20)]
        public string Name { get; set; }

        [Column(DbType = "TEXT", Name = "remark")]
        public string Remark { get; set; }

        [Column(Name = "unique")]
        public int Unique { get; set; }

        [Column(Name = "isBasicData")] public bool IsBasicData { get; set; }
    }
}

File: act/act-modules/act-modules-zero/src/main/java/com/act/modules/zero/internal/domain/SysOperate.java
package com.act.modules.zero.internal.domain;

import com.act.core.domain.BaseEntity;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

@Data
@TableName("sys_operate")
@SuppressWarnings("all")
public class SysOperate extends BaseEntity<Long> {
    @TableField("name")
    private String name;
    @TableField("`unique`")
    private int unique = 1001;
    @TableField("isBasicData")
    private Boolean isBasicData = false;
    @TableField("remark")
    private String remark;
}
